### PR TITLE
[SG-39363] Extract bootstrap style import` @import 'bootstrap/scss/card'` in `client/vscode`

### PR DESCRIPTION
**Pull request by gitstart-sourcegraph**
## Description
Removes bootsrap `card` stylesheet import and converted to wildcard component

## Ref
[Gitstart Issue
](https://app.gitstart.com/clients/sourcegraph/tickets/SG-39363)[Sourcegraph Issue](https://github.com/sourcegraph/sourcegraph/issues/39363)

## Success criteria
1. All use cases of `card-*` are migrated to use wildcard's `Card` component.

## Implementation details
1.  Remove bootstrap import @import 'card/scss/card' in `forkedBranded.scss`.
2. Covert use cases of `card-*` className to `Card` wildcard component.
3. If step 2. above is not doable, let's copy style rule from bootstrap to be used internally.

## Test Plan
- Run the vscode app on this branch
- See all use cases of bootstrap's `card-*` className on the codebase (seam to be only in `HomeFooter.tsx`)
- Confirm there are no UI diffs.

## App preview:

- [Web](https://sg-web-contractors-sg-39363.onrender.com)
- [Storybook](https://5f0f381c0e50750022dc6bf7-ejelitfaoe.chromatic.com)

Check out the [client app preview documentation](https://docs.sourcegraph.com/dev/how-to/client_pr_previews) to learn more.

### CODE REVIEW

File: client/vscode/src/webview/search-panel/components/HomeFooter.tsx [hunk 1]
```diff
@@ -48,9 +49,9 @@ const SearchExamples: React.FunctionComponent<React.PropsWithChildren<SearchExam
             <div className={styles.searchExamples}>
                 {examples.map(example => (
                     <div key={example.queryPreview} className="search-example-card-wrapper">
-                        <button
-                            type="button"
-                            className={classNames('card p-0 w-100', styles.searchExampleCard)}
+                        <Card
+                            as="button"
+                            className={classNames('p-0 w-100', styles.searchExampleCard)}
```
Comment (gitstart-sourcegraph) [inline client/vscode/src/webview/search-panel/components/HomeFooter.tsx:54]: A side effect of migrations to an interactive card: we now have `hover/focus` ring shadow introduced similar to [use case here](https://storybook.sgdev.org/?path=/story/wildcard-card--interactive-card), which was not visible in existing UI.
Would love to get your thoughts on wether or not this is a desired behavior.
CC: @valerybugakov

Comment (valerybugakov) [reply client/vscode/src/webview/search-panel/components/HomeFooter.tsx:54]: @gitstart-sourcegraph, could you attach the screenshot?

Comment (gitstart-sourcegraph) [reply client/vscode/src/webview/search-panel/components/HomeFooter.tsx:54]: @valerybugakov yeah, these are screenshots of before and after these changes

Hover State Before
Screenshot 2022-07-25 at 5 00 28 PM

Hover State After
Screenshot 2022-07-25 at 5 00 17 PM

Comment (valerybugakov) [reply client/vscode/src/webview/search-panel/components/HomeFooter.tsx:54]: It seems that this is the correct behavior of the button card. 
Could you fix the issue with the icon background going over the border?

Screen Shot 2022-07-27 at 13 31 18

**Review by valerybugakov — APPROVED** [review]
Thanks!

**Pull request merged**